style.css

```diff
@@ -500,7 +706,20 @@ body {
   grid-template-rows: repeat(6, 470px);
   justify-items: center;
   height: 2972px;
-  gap: 24px;
+  row-gap: 24px;
+}
+
+@media (min-width: 992px) {
+  .works-cards {
+    grid-template-columns: repeat(3, 31%);
+    grid-template-rows: repeat(2, 50%);
+    height: 988px;
+    width: 100%;
+    align-content: center;
+    justify-content: center;
+    justify-items: normal;
+    column-gap: 20px;
+  }
```

Comment:
- [ ] You have implemented the projects' cards in a good way 👍 . It would be better to improve the works section and all the sections (About, contact) by increasing the left and right margins to improve the UI design and also to match the Figma design.
- your design:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/77190203/154509982-c2c77177-8d27-4326-bc58-323f3c161acf.png">
Figma design:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/77190203/154510455-1eeeecfc-424c-40c9-b52b-9a0f3110a3bc.png">